lib/src/asn1/liblte_mme.cc

```diff
@@ -3019,6 +3020,10 @@ LIBLTE_ERROR_ENUM liblte_mme_unpack_emergency_number_list_ie(uint8**
     emerg_num_list->N_emerg_nums = 0;
     while (length < sent_length) {
       idx                                               = emerg_num_list->N_emerg_nums;
+      if(idx >= LIBLTE_MME_EMERGENCY_NUMBER_LIST_MAX_SIZE) {
+        srsran::console("Error: The length of emergency number list is too long!\n");
```

Comment:
I am sorry, checking the code again it seems we are not really handling other unpacking errors with a console print. In order to keep it consistent, just do the return there and that's it.